Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -15,6 +15,7 @@ protocol AutocompleteTextFieldDelegate: class {
     func autocompleteTextFieldShouldClear(_ autocompleteTextField: AutocompleteTextField) -> Bool
     func autocompleteTextFieldDidBeginEditing(_ autocompleteTextField: AutocompleteTextField)
     func autocompleteTextFieldDidCancel(_ autocompleteTextField: AutocompleteTextField)
+    func autocompletePasteAndGo( autocompleteTextField: AutocompleteTextField)
```

Comment:
Not sure if there should be a `_` to make it `func autocompletePasteAndGo(_ autocompleteTextField: AutocompleteTextField)`